### DISCUSSION TITLE
Accordion icon

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#1819] [Proposals]
### Goal

Visual consistency between select component and accordion component

### Context

We are using different types of "chevron" icon around the system. 

- Select
- Accordion
- Lists
- ButtonLink (with href)

### Impact

Improve the association of dropdown elements for users

### Discussion Questions

What about lists? should be the same chevron?

In my opinion, not necessary, because the goal of the list chevrons is totally different

**Comment by AnaMontes11** [comment]
Closing this discussion since it's already been handled in issue #2142 Chevron size normalization.
